tests/migration_test.go

```diff
@@ -1011,6 +1020,78 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				By("Waiting for NFS pod to disappear")
 				tests.WaitForPodToDisappearWithTimeout(tests.NFSTargetName, 120)
 			})
+
+			It(" should be migrated successfully, using guest agent on VM w/ postcopy", func() {
+				data := map[string]string{
+					"migrationMode":           "PostCopy",
```

Comment:
ah, I guess this needs to be adjusted too.
Otherwise, the migration is being canceled.:

```
{"component":"virt-launcher","level":"error","msg":"operation aborted: migration out job: canceled by client","pos":"qemuMigrationJobCheckStatus:1563","subcomponent":"libvirt","thread":"34","timestamp":"2020-09-24T09:30:03.202000Z"}
```